cmd/oc-mirror/main.go

```diff
@@ -1,11 +1,36 @@
 package main
 
 import (
+	"bytes"
+	"embed"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"slices"
+	"syscall"
+
 	"github.com/openshift/oc-mirror/pkg/cli/mirror"
+	"k8s.io/klog/v2"
 	kcmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
+//go:embed data/*
+var mirrorv2 embed.FS
+
 func main() {
+	if isV2() {
+		if err := runOcMirrorV2Cmd(os.Args); err != nil {
+			klog.Errorf("%s", err.Error())
```

Comment:
Need to experiment with this a bit more but I think we just want to print the error if it's not an `exec.ExitError`.